buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1436,4 +1433,31 @@ def create_job(JOB_NAME, SDK_VERSION, SPEC, downstreamJobType, id) {
         3, 120)
 }
 
+def set_build_variables_per_node() {
```

Comment:
This only gets executed on the nodes themselves correct, not on master?

Reply:
To answer my own question based on a review of your PR build, yes it only executes on the node in the build job, not the higher level pipeline jobs.

Reply:
correct